packages/perseus/src/validation.types.ts

```diff
@@ -82,17 +81,26 @@ export type PerseusGradedGroupRubric = PerseusGradedGroupWidgetOptions;
 
 export type PerseusGradedGroupSetRubric = PerseusGradedGroupSetWidgetOptions;
 
-export type PerseusGrapherRubric = PerseusGrapherWidgetOptions;
+export type PerseusGrapherRubric = {
+    correct:
+        | {type: "absolute_value"; coords: [Coord, Coord]}
+        | {
+              type: "exponential";
+              asymptote: [Coord, Coord];
+              coords: [Coord, Coord];
+          }
+        | {type: "linear"; coords: [Coord, Coord]}
+        | {type: "logarithm"; asymptote: [Coord, Coord]; coords: [Coord, Coord]}
+        | {type: "quadratic"; coords: [Coord, Coord]}
+        | {type: "sinusoid"; coords: [Coord, Coord]}
+        | {type: "tangent"; coords: [Coord, Coord]};
+};
```

Comment:
Came up with a couple other ways to do this. I don't _think_ either of these are preferred, and since it works as is, just wanted to leave it. But if either of the options below work better, would love to know :)

Using an enum _(Thanks @anakaren-rojas for showing me this!)_ -
<img src="https://github.com/user-attachments/assets/fe507d96-bdb9-4d2a-9276-4d8d4cc613b3" height=400/>

Using as const with object types - 
<img src="https://github.com/user-attachments/assets/25352063-4b6a-4381-8449-dda22308bb19" height=400/>

Reply:
That way you did it is correct (except that I think we should share a type instead of copying it). We try not to use enums and in TS I haven't noticed much of a difference between `TypeObject.Linear` and `"linear"`; that's more of something you'd see in Java.